src/adr/articles/0002-verifiable-credentials.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: layouts/fullwidth.njk
+title: "2. Verifiable Credentials"
+weight: 1
+date: 2021-07-02
+date_view: "2021-07-02"
+review_in: 3 months
+tags: adr
+areas_of_coverage: "Verifiable Credentials"
+status: proposed
+contributors: "John Nolan"
+---
+# {{title}}
+
+## Overview
+
+**Date**: {{date_view}}
+
+### Status
+
+<strong class="govuk-tag">{{status}}</strong>
+
+### Contributors
+
+{{contributors}}
+
+### Areas of coverage
+
+{{areas_of_coverage}}
+
+## Context
+
+We want to give the Citizen a way of using a verifiable digital representation of a LPA. This will enable them to use this to prove they are who they say they are.
+
+If an LPA is no longer a paper deed, then this is one solution to what a digital version could be.
+
+This could be an optional addition to the existing Use a LPA service to allow for a wider demographic of users.
+
+It needs to be fully interopable with
+
+1. Future technology such as mobile phone digital wallets
+2. Industry standard validation and authenticity implementations
+3. Potential for other agencies to use as a proof of identity
+
+We would need to be able to reliably sign the credential for others to validate the authenticity.
+
+We would need the Verifiable Credential to be able to store the appropriate required information for the consumer to understand its responsibilities.
+
+## Technical
+
+### Interoperability - How does this enable the exchange of information
+
+The Verifiable Credentials data model is a [W3C Open Standard](https://www.w3.org/TR/vc-data-model/). As long as the standards are adhered to, interopability will be high.
```

Comment:
But the standard doesn't recommend a single way to create the proof or represent the proof, so it will depend on what we choose vs. wallets/etc decide to implement.